data/scripts/get_tract_districts.py

```diff
@@ -9,6 +9,7 @@
 
 centroids = []
 
+# get gemoetries for each US Census Tract in Illinois
```

Comment:
Typo, "gemoetries" to "geometries", but I'll leave it 😆

Reply:
fixed